pages/configuration/server.conf.rst

```diff
@@ -0,0 +1,407 @@
+* ``output_flush_interval = 1``
+    * Flush interval (in seconds) for the Elasticsearch output. This is the maximum amount of time between two batches of messages written to Elasticsearch. It is only effective at all if your minimum number of messages for this time period is less than ``output_batch_size * outputbuffer_processors``.
+
+* ``output_fault_count_threshold = 5``
+* ``output_fault_penalty_seconds = 30``
+    * As stream outputs are loaded only on demand, an output which is failing to initialize will be tried over and over again. To prevent this, the following configuration options define after how many faults an output will not be tried again for an also configurable amount of seconds.
+* ``processbuffer_processors = 5``
+* ``outputbuffer_processors = 3``
+    * The number of parallel running processors.
+    * Raise this number if your buffers are filling up.
+
+
+* ``outputbuffer_processor_keep_alive_time = 5000``
+* ``outputbuffer_processor_threads_core_pool_size = 3``
+* ``outputbuffer_processor_threads_max_pool_size = 30``
+* ``udp_recvbuffer_sizes = 1048576``
+    * UDP receive buffer size for all message inputs (e. g. SyslogUDPInput).
+
+* ``processor_wait_strategy = blocking``
+    * Wait strategy describing how buffer processors wait on a cursor sequence. (default: sleeping)
+    * Possible types:
+        - ``yielding`` - Compromise between performance and CPU usage.
+        - ``sleeping`` - Compromise between performance and CPU usage. Latency spikes can occur after quiet periods.
+        - ``blocking`` -  High throughput, low latency, higher CPU usage.
+        - ``busy_spinning`` - Avoids syscalls which could introduce latency jitter. Best when threads can be bound to specific CPU cores.
+* ``ring_size = 65536``
+    * Size of internal ring buffers. Raise this if raising ``outputbuffer_processors`` does not help anymore.
+    * For optimum performance your LogMessage objects in the ring buffer should fit in your CPU L3 cache.
+    * Must be a power of 2. (512, 1024, 2048, ...)
+* ``inputbuffer_ring_size = 65536``
+* ``inputbuffer_processors = 2``
+* ``inputbuffer_wait_strategy = blocking``
+* ``message_journal_enabled = true``
+    * Enable the disk based message journal.
+
+* ``message_journal_dir = data/journal``
+      * The directory which will be used to store the message journal. The directory must me exclusively used by Graylog and must not contain any other files than the ones created by Graylog itself.
+
+  .. attention:: If you create a seperate partition for the journal files and use a file system creating directories like 'lost+found' in the root directory, you need to create a sub directory for your journal. Otherwise Graylog will log an error message that the journal is corrupt and Graylog will not start.
+* ``message_journal_max_age = 12h``
+* ``message_journal_max_size = 5gb``
```

Comment:
You could add here that the dropped message will be the first inserted and not some random.

Reply:
Can you clarify this?  
Is the suggestion to add another note saying something like:
> When the journal is full and it keeps receiving messages, it will start dropping messages as a FIFO queue: The first dropped message will be the first inserted and so on (and not some random).

Reply:
yes @martinbydefault you got my point